app/javascript/components/analytics/display/WorkflowTable.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import InfoTooltip from '../../util/InfoTooltip';
+
+class WorkflowTable extends React.Component {
+  render() {
+    return (
+      <React.Fragment>
+        <div className="analytics-table-header">
+          {this.props.title}
+          {this.props.tooltipKey && (
+            <span className="h6 text-left">
+              <InfoTooltip tooltipTextKey={this.props.tooltipKey} location="right"></InfoTooltip>
+            </span>
+          )}
+        </div>
+        <table className="analytics-table text-right">
+          <thead>
+            <tr className="header">
+              <th></th>
+              <th>Exposure</th>
+              <th>Isolation</th>
```

Comment:
This makes the assumption that the `Isolation` workflow will always be present. How do you feel about making this a configurable option above? Maybe `workflows = ['Exposure', 'Isolation']` and iterating over them?

Reply:
done: https://github.com/SaraAlert/SaraAlert/pull/993/commits/0f40f134cd1296ff028fa90890bad2bcb661f905